tests/testthat/test-correlation.R

```diff
@@ -36,6 +36,40 @@ test_that('correlation works with two data tables', {
 
 })
 
+test_that("correlation works with a single data table", {
+  nSamples = 200
+  testData1 <- data.table(
+    "contA1" = rnorm(nSamples),
+    "contB1" = rnorm(nSamples),
+    "contC1" = rnorm(nSamples)
+  )
+  corrResult <- correlation(testData1, method='spearman', verbose=F)
```

Comment:
see other comment in the correlation method, but `nrow(corrResult)` should be equal to `ncol(testData1)*(ncol(testData1) - 1)/2`